src/layouts/AuthLayout/AuthLayout.module.scss

```
@@ -0,0 +1,30 @@
+@use '@/styles/variables' as *;
+
+.container {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  box-sizing: border-box;
+  min-height: 100vh;
+  padding: 0 16px;
+  padding-top: 56px;
```

```suggestion
  padding-top: 56px;
```
이건 헤더 영역을 임시로 표시하기 위함일까요..?